latch_cli/services/register/register.py

```diff
@@ -237,9 +242,41 @@ def register(
         disable_auto_version=disable_auto_version,
         remote=remote,
     ) as ctx:
-        print(f"Initializing registration for {pkg_root}")
+        assert ctx.workflow_name is not None, "Unable to determine workflow name"
+        assert ctx.version is not None, "Unable to determine workflow version"
+
+        # todo(maximsmol): we really want the workflow display name here
+        click.echo(
+            " ".join(
+                [click.style("Workflow name:", fg="bright_blue"), ctx.workflow_name]
+            )
+        )
+        click.echo(" ".join([click.style("Version:", fg="bright_blue"), ctx.version]))
+
+        workspaces = get_workspaces()
+        ws_name = next(
+            (x[1] for x in workspaces.items() if x[0] == current_workspace()), "N/A"
```

Comment:
have a check if `current_workspace() == ""` and default to the personal workspace